newIDE/app/src/SceneEditor/index.js

```diff
@@ -2146,6 +2197,11 @@ export default class SceneEditor extends React.Component<Props, State> {
                         objectGroup
                       );
                     }
+                    // TODO Set the `global` attribute correctly.
```

Comment:
Can we either do this or not pass global: true/false at all if we don't know or if not useful?

Reply:
It sounds to be a complicated refactor. I prefer to do it later.